src/bindings/c/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ set(C_FORTRAN_LIB_SRCS
     growtree.f90
     indices.f90
     imports.f90
+    precision.f90
```

Comment:
No need to expose precision to python bindings, only needed when someone will run routines from the python front end. Can delete this line and file.